### PR TITLE
[1263] fix: cron multiple execution, #1261 cron error message

### PR DESCRIPTION
**Pull request by jremen**
**Changes proposed in this Pull Request**
Fixed cron multiple executions due to setInterval, error message if status not in 200-299 range

Close QualityUnit/web-issues#1256
Close QualityUnit/web-issues#1259
Close QualityUnit/web-issues#1261
Close QualityUnit/web-issues#1262
Close QualityUnit/web-issues#1263
Close QualityUnit/web-issues#1265
Close QualityUnit/web-issues#1266

### CODE REVIEW

File: admin/src/hooks/filteringSorting.jsx [hunk 1]
```diff
@@ -45,16 +45,16 @@ export function useFilter( { slug } ) {
...
 export function useSorting( { slug } ) {
 	const [ sortingColumn, setSortingColumn ] = useState( '' );
-	const [ tableQuery, setTableQuery ] = useState( {} );
-	useEffect( () => {
-		get( slug ).then( ( query ) => {
-			setTableQuery( query );
-		} );
-	}, [ slug ] );
+	// const [ tableQuery, setTableQuery ] = useState( {} );
+	// useEffect( () => {
```
Comment (vzeman) [inline admin/src/hooks/filteringSorting.jsx:49]: do we need these comments?

File: admin/src/hooks/filteringSorting.jsx [hunk 2]
```diff
@@ -3,16 +3,16 @@ import { get, set } from 'idb-keyval';
...
-	useEffect( () => {
-		get( slug ).then( ( query ) => {
-			const q = query?.currentFilters;
-			setUrl( Object.keys( q ).length ? q : {} );
-			setTableQuery( query );
-		} );
-	}, [ slug ] );
+	// useEffect( () => {
+	// 	get( slug ).then( ( query ) => {
+	// 		const q = query?.currentFilters;
```
Comment (vzeman) [inline admin/src/hooks/filteringSorting.jsx:11]: do we need these comments in the future?

**Review by vzeman — CHANGES_REQUESTED** [review]

File: admin/src/hooks/filteringSorting.jsx (hunk 1 above)
Comment (jremen) [reply admin/src/hooks/filteringSorting.jsx:49]: theoretically, no

File: admin/src/hooks/filteringSorting.jsx (hunk 2 above)
Comment (jremen) [reply admin/src/hooks/filteringSorting.jsx:11]: will change to other behaviour later

**Review by vzeman — APPROVED** [review]

**Pull request merged**